README.md

```diff
@@ -1,23 +1,36 @@
+# DocTester
 
-Markdown file tester
-# this is a sample file to test feature.
-## POC file
+This library is used to run code samples in markdown documents as acceptance tests, using simple comment-based assertions. By testing code snippets in documentation, you can have greater confidence that code in your documentation works as advertised.
 
-```js
-import { add } from './add';
-add(4, 5) // equals: 9;
-add(4, 5) // not-equals: 6;
-add(2, 3) // throws;
+## Installation
+```sh
+npm install sparshithNR/doc-tester#master --save-dev
```

Comment:
you should likely just do `yarn add doc-tester` or `npm install doc-tester` and release the project to NPM.